uvloop/includes/stdlib.pxi

```diff
@@ -63,6 +63,8 @@ cdef gc_disable = gc.disable
 cdef iter_chain = itertools.chain
 cdef inspect_isgenerator = inspect.isgenerator
 
+cdef int IPV6_V6ONLY = getattr(socket, 'IPV6_V6ONLY')
+cdef int has_SO_REUSEADDR = os.name == 'posix' and sys.platform != 'cygwin'
```

Comment:
This is directly lifted from: https://github.com/python/cpython/blob/3e028b2d40370dc986b6f3146a7ae927bc119f97/Lib/asyncio/base_events.py#L1220

Reply:
It's not about the platform having `SO_REUSEADDR` or not, its about it behaving quite differently on Windows.  Coincidentally there's a relevant discussion about this option here: https://github.com/python/cpython/pull/11784#discussion_r254843057 if you are interested what's going on.

Regarding `has_SO_REUSEADDR` -- I'd just inline this check to `create_server()`.

Reply:
(warning: the linked discussion is a bit... "inflammatory" 🤨)

Reply:
I'm not sure how to inline the check, I had done so initially but the code bombed, even if I `import os` and `import sys` I don't seem to be able to use `os.name` or `sys.platform`.. some Cython pecularity I don't know?

Reply:
just add `os_name = os.name` and `sys_platform = sys.platform` to `uvloop/includes/stdlib.pxi`.  We'll later than be able to use search/replace to remove that file.